.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
 max-line-length = 119
```

Comment:
Oh yes, let me remove that. It is from one of my previous comment. i was trying to add a static pdf file. and was having issues because flake8 was trying to scan it for formatting and syntax